src/components/NewBookEntry.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const NewBookEntry = () => (
+  <div className=" newbookentry">
+    <h4>Add New Book</h4>
+    <form className="frm">
+      <input
+        className="newbk"
+        type="text"
+        placeholder="Add a new book..."
+        id="newbook"
+        name="bookname"
+      />
+      <select className="author">
+        <option readOnly>Author</option>
+        <option>abc</option>
+        <option>xyz</option>
+      </select>
```

Comment:
- [ ] In the project's requirement, we must use the input to get the author. Kindly get the author with input. You can keep your selection for the genres of your book.